cogs/StaffRequirements.py

```diff
@@ -60,7 +60,9 @@ async def checkreqs(self, ctx: commands.Context) -> None:
         time_since_creation = (
             datetime.now(tz=account.created_at.tzinfo) - account.created_at
         )
-        time_since_join = datetime.now(tz=account.joined_at.tzinfo) - account.joined_at
+        time_since_join = (
+            datetime.now(tz=ctx.guild.me.joined_at.tzinfo) - ctx.guild.me.joined_at
```

Comment:
Why `ctx.guild.me.joined_at`? That will be the time that the bot joined, not the time that the user joined